landoscript/tests/conftest.py

```diff
@@ -217,6 +217,19 @@ def assert_add_commit_response(action, commit_msg_strings, initial_values, expec
 
     diffs = action["diff"].split("\ndiff")
 
+    # Extract file paths from diffs and verify they are sorted
+    file_paths = []
+    for diff in diffs:
+        if not diff:
+            continue
+
+        path_line = diff.split("\n")[0]
+        if path_line.startswith("diff --git"):
```

Comment:
Unfortunately, this misses all but the first file. Due to the way the diffs are split, we lose the `diff` in the in the subsequent sections.

You can probably fix this by adjusting `diffs` after the splitting happens.